spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/cassandra/CassandraDataAutoConfiguration.java

```diff
@@ -80,10 +113,41 @@ public CassandraSessionFactoryBean session(CassandraConverter converter)
 		session.setCluster(this.cluster);
 		session.setConverter(converter);
 		session.setKeyspaceName(this.properties.getKeyspaceName());
-		session.setSchemaAction(SchemaAction.NONE);
+		session.setSchemaAction(SchemaAction.valueOf(this.properties.getSchemaAction()));
```

Comment:
So I'll change `schemaAction` to `org.springframework.data.cassandra.config.SchemaAction`. #4890 becomes to a blocker for this PR?